pyproject.toml

```diff
@@ -9,6 +9,7 @@ python = "==3.8.10"
 Flask = "==2.0.3"
 python-dotenv = "==0.19.2"
 Flask-WTF = "==1.0.0"
+Flask-Bootstrap = "^3.3.7"
```

Comment:
Can you pin this? Also, I ran poetry install and received a warning about an outdated lock file. you can update the lock file by running 'poetry update poetry.lock'

Reply:
I think I've resolved this. Thanks for the heads-up.